src/Components/Server/src/Circuits/CircuitClientProxy.cs

```diff
@@ -49,5 +47,10 @@ public Task SendCoreAsync(string method, object[] args, CancellationToken cancel
 
             return Client.SendCoreAsync(method, args, cancellationToken);
         }
+
+        internal static CircuitClientProxy CreateOffline()
+        {
+            return new CircuitClientProxy();
+        }
```

Comment:
Is this really useful to have if it's an alias for the *public* default constructor? This seems like a stylistic choice that *doesn't* make things clearer...

Reply:
It felt akward going from CircuitClientProxy.Offline to new CircuitClientProxy(). The first one felt more descriptive, that's what I went for it. I don't feel strongly about it in any case, so I'll just use the ctor.

Reply:
changed